### PR TITLE
L2D4A4

### PR DESCRIPTION
**Pull request by evyryan**

### CODE REVIEW

File: kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml [hunk 1]
```diff
@@ -70,6 +70,7 @@
    </p>
             <div xml:id="kzz7yh-f16299-Dd1e124">
                <head xml:id="kzz7yh-f16299-Hd1e126">Quaestio 1</head>
+               <head type="question-title">Utrum mali angeli ante suum casum habuerunt cornitione matutinam</head>
```
Comment (jeffreycwitt) [inline kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml:73]:
```suggestion
               <head type="question-title">Utrum mali angeli ante suum casum habuerunt cognitione matutinam</head>
```

File: kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml [hunk 2]
```diff
@@ -114,7 +115,7 @@
    </p>
                <p xml:id="kzz7yh-f16299-d1e204">
                   <lb ed="#V" n="74"/>Contra Dyosminus 4. ca. de diui. no inter medium et finem
```
Comment (jeffreycwitt) [inline kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml:117]:
```suggestion
                  <lb ed="#V" n="74"/>Contra Dionysius 4. ca. de diui. no. inter medium et finem 
```

File: kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml [hunk 3]
```diff
@@ -167,7 +168,7 @@
     </p>
                <p xml:id="kzz7yh-f16299-d1e300">
                   <lb ed="#V" n="111"/>Ad primum in oppositum. Et secundum. Et tertium 
-      <lb ed="#V" n="112"/>dicendum quod ille auctoritates 
+      <lb ed="#V" n="112"/>dicendum <!--?--> quod ille auctoritates
```
Comment (jeffreycwitt) [inline kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml:171]: dicendum is correct 

```suggestion
      <lb ed="#V" n="112"/>dicendum quod ille auctoritates 
```

File: kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml [hunk 4]
```diff
@@ -177,10 +178,11 @@
             </div>
             <div xml:id="kzz7yh-f16299-Dd1e316">
                <head xml:id="kzz7yh-f16299-Hd1e318">Quaestio 2</head>
+               <head type="question-title">Utrum boni angeli post suam conversionem habuerunt congitionem vespertinam</head>
```
Comment (jeffreycwitt) [inline kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml:181]:
```suggestion
               <head type="question-title">Utrum boni angeli post suam conversionem habuerunt cognitionem vespertinam</head>
```

File: kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml [hunk 5]
```diff
@@ -177,10 +178,11 @@
...
                <p xml:id="kzz7yh-f16299-d1e320">
       ¶ Quaestio II. 
-      <lb ed="#V" n="115"/>SEcundo queritur vtrum boni angeli potest 
-      <lb ed="#V" n="116"/>sua conuersionem habuerunt 
+      <lb ed="#V" n="115"/>SEcundo queritur vtrum boni angeli potest <!--Post vs Potest--> 
+      <lb ed="#V" n="116"/>suam conuersionem habuerunt
```
Comment (jeffreycwitt) [inline kzz7yh-f16299/cod-ve09v2_kzz7yh-f16299.xml:185]:
```suggestion
      <lb ed="#V" n="115"/>SEcundo queritur vtrum boni angeli post
      <lb ed="#V" n="116"/>suam conuersionem habuerunt 
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]

**Comment by evyryan** [comment]
I don't know what I just did. If i was supposed to close it or not. Sorry

**Pull request closed without merging**